oteapi/datacache/datacache.py

```diff
@@ -225,34 +178,35 @@ def getfile(  # pylint: disable=too-many-arguments
         """Write the value for `key` to file and return the filename.
 
         The file is created in the default directory for temporary
-        files (which can be controlled by the `TEMPDIR`, `TEMP` or `TMP`
+        files (which can be controlled by the TEMPDIR, TEMP or TMP
         environment variables). It is readable and writable only for
         the current user.
 
+        This method is intended to be used in a `with` statement, to
+        automatically delete the file when leaving the context.
+
```

Comment:
```suggestion
```

Reply:
What about moving this sentence and the example down below the argument description. 

numpy docstrings has Notes and Examples as standard sections. Is there a similar standard in google docstrings?